lib/content-services/src/lib/content-metadata/components/content-metadata-card/content-metadata-card.component.spec.ts

```diff
@@ -34,15 +45,40 @@ describe('ContentMetadataCardComponent', () => {
     let node: Node;
     const preset = 'custom-preset';
     let nodeAspectService: NodeAspectService = null;
+    let tagService: TagService = null;
+    let categoryService: CategoryService = null;
+    let propertyDescriptorsService: PropertyDescriptorsService = null;
 
     const getToggleEditButton = () => fixture.debugElement.query(By.css('[data-automation-id="meta-data-card-toggle-edit"]'));
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [ContentTestingModule]
+            imports: [
```

Comment:
this does not make any sense

Reply:
It does actually, all that stuff that was embedded in the ContentTEstingModule was hiding the fact that this unit test was trying to do real REST calls, which is an issue that was at a certain point blocking the execution of the whole chain. 
As this was the only way to find out, it's working and it is stable now so i consider this an improvement actually, but feel free to change it as you wish after we merged the migration for ng15